sources/packages/backend/apps/db-migrations/src/sql/SFASRestrictionMaps/Create-sfas-restriction-maps.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE sims.sfas_restriction_maps (
+  id SERIAL PRIMARY KEY,
+  legacy_code VARCHAR(4) NOT NULL,
+  code VARCHAR(10) NOT NULL,
+  is_legacy_only BOOLEAN NOT NULL,
+  -- Audit columns
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  UNIQUE (legacy_code, code)
```

Comment:
Can we name the constraint.?

We have been following this pattern looking at recent code.

![image](https://github.com/user-attachments/assets/14fc897d-aabd-435a-86d9-3278b3444c87)

Reply:
Are you talking about the name convention or about executing it in an alter table?

Reply:
About convention of `CONSTRAINT constraint_name  UNIQUE (legacy_code, code)`

Reply:
As we talked, we will be keeping this as it is.